package.json

```diff
@@ -36,6 +36,7 @@
     "start-clock": "node --max_old_space_size=1500 --nouse-idle-notification clock/index.js",
     "start-worker": "node --max_old_space_size=1500 --nouse-idle-notification worker/jobProcessor.js",
     "test-api": "mocha -R dot --recursive tests/api",
+    "test-api-log": "ENABLE_API_LOGS=true mocha -R dot --recursive tests/api",
```

Comment:
I like the blanket coverage by doing it this way, but it also means these tests are not counted by coveralls. Can you think of another way of achieving this? We've added a test utility function called "toggleOverride" which lets us turn a feature on or off in a test's before/after function (as long as the feature doesn't require re-initializing top-level app server index.js).